reactivesocket-client/src/main/java/io/reactivesocket/client/ClientBuilder.java

```diff
@@ -219,5 +219,15 @@ public void onNext(Collection<T> socketAddresses) {
             null
         );
     }
+
+    @Override
+    public String toString() {
+        return "ClientBuilder("
+            + "source=" + source.toString()
```

Comment:
I know that is safe, but the current null.toString() in the diff is unsafe.  There is a bizarro parallel universe, where Steve intended a NPE if you call toString with nulls.